UltraStar Play/Assets/Scenes/Sing/SingSceneController.cs

```diff
@@ -329,7 +329,13 @@ private MicProfile GetDefaultMicProfile()
 
     private SongMeta GetDefaultSongMeta()
     {
-        return SongMetaManager.Instance.SongMetas.First();
+        IEnumerable<SongMeta> defaultSongMetas = SongMetaManager.Instance.SongMetas.Where(it => it.Title == defaultSongName);
```

Comment:
How is this supposed to work? Except you, probably no one has that song in their dev setup...

Reply:
The defaultSongName can be set in the Inspector of the SongSceneController because it is a public member. For debugging, I expect that one wants to load a specific song when starting the SingScene in the Unity Editor. At least I use this.